Jenkinsfile

```diff
@@ -0,0 +1,95 @@
+/* groovylint-disable DuplicateStringLiteral */
+/* groovylint-disable-next-line CompileStatic */
+pipeline {
+    agent any
+    options {
+        buildDiscarder logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '5', numToKeepStr: '5')
+        disableConcurrentBuilds()
+    }
+    stages {
+        stage('Test Jenkins Out') {
+            agent any
+            steps {
+                echo 'CI Triggered...'
+            }
+        }
+        stage('ci') {
+            agent {
+                docker {
+                    image 'node:18-alpine'
+                    args '-u root'
+                }
+            }
+            environment {
+                NODE_ENV = 'test'
+                NODE_PORT = 3131
+                ARANGO_DB_HOST_DOMAIN = 'arangotest.digiteched.com'
+                ARANGO_DB_HOST_SCHEME = 'https'
+                ARANGO_DB_HOST_PORT = 443
+                ARANGO_DB_ROOT_PASSWORD = credentials('arango-test-root-password')
+                ARANGO_DB_USER = 'dbuser'
+                ARANGO_DB_USER_PASSWORD = credentials('arango-test-db-user-password')
+                ARANGO_DB_NAME = 'staging'
+                AUTH0_ISSUER_URL = 'https://foo.auth0.com/'
```

Comment:
Is it possible to use ENV vars here? I'm guessing I'm missing a detail of how this file works.

Reply:
This is how you set environment variables in an ephemeral Docker container. My other option, I suppose, is to make a managed .env config file with Jenkins.  I'll look at that when I am cleaning up the Jenkinsfile.